packages/flutter/test/foundation/diagnostics_test.dart

```diff
@@ -274,29 +274,16 @@ void main() {
       'dense',
       style: DiagnosticsTreeStyle.dense,
       golden:
-      'TestTree#00000\n'
-      '│stringProperty1: value1\n'
-      '│doubleProperty1: 42.5\n'
-      '│roundedProperty: 0.3\n'
-      '│nullProperty: null\n'
-      '│<root node>\n'
-      '│\n'
-      '├child node A: TestTree#00000\n'
-      '├child node B: TestTree#00000\n'
-      '││p1: v1\n'
-      '││p2: v2\n'
-      '││\n'
-      '│├child node B1: TestTree#00000\n'
-      '│├child node B2: TestTree#00000\n'
-      '││ property1: value1\n'
-      '│└child node B3: TestTree#00000\n'
-      '│  <leaf node>\n'
-      '│  foo: 42\n'
-      '└child node C: TestTree#00000\n'
-      '  foo:\n'
-      '  multi\n'
-      '  line\n'
-      '  value!\n',
+        'TestTree#00000(stringProperty1: value1; doubleProperty1: 42.5; roundedProperty: 0.3; nullProperty: null; <root node>)\n'
```

Comment:
I had misunderstood how the dense style should look and the dense widget tree was working previously by adding all the properties on the side to the description instead of through the standard property syntax.